sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -595,6 +631,76 @@ case class Inline(child: Expression) extends UnaryExpression with CollectionGene
   override protected def withNewChildInternal(newChild: Expression): Inline = copy(child = newChild)
 }
 
+trait InlineGeneratorBuilderBase extends GeneratorBuilder {
+  override def functionSignature: Option[FunctionSignature] =
+    Some(FunctionSignature(Seq(InputParameter("input"))))
+  override def buildGenerator(funcName: String, expressions: Seq[Expression]): Generator = {
+    assert(expressions.size == 1)
+    Inline(expressions(0))
+  }
+}
+
+// scalastyle:off line.size.limit line.contains.tab
+@ExpressionDescription(
+  usage = "_FUNC_(expr) - Explodes an array of structs into a table. Uses column names col1, col2, etc. by default unless specified otherwise.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_(array(struct(1, 'a'), struct(2, 'b')));
+       1	a
+       2	b
+      > SELECT _FUNC_(input => array(struct(1, 'a'), struct(2, 'b')));
+       1	a
+       2	b
+  """,
+  since = "2.0.0",
+  group = "generator_funcs")
+// scalastyle:on line.size.limit line.contains.tab
+object InlineExpressionBuilder extends ExpressionBuilder {
+  override def functionSignature: Option[FunctionSignature] =
+    Some(FunctionSignature(Seq(InputParameter("input"))))
```

Comment:
should this be

```suggestion
    Some(FunctionSignature(Seq(InputParameter("collection"))))
```
?

Reply:
cc @dtenedor